Swift.gitignore

```diff
@@ -32,6 +36,12 @@ xcuserdata/
 timeline.xctimeline
 playground.xcworkspace
 
+## Finder
+.DS_Store
```

Comment:
This is an OS-specific rule, and covered by `Global/macOS.gitignore`. Please exclude it from this PR.